posts/2019/07/ibis-python-data-analysis-productivity-framework.ipynb

```diff
@@ -0,0 +1,424 @@
+    "\n",
+    "Ibis was created by Wes McKinney (@wesm) and is mainly maintained by Phillip Cloud (@cpcloud) and Krisztián Szűcs (kszucs). Also, recently, I was invited as contributor to Ibis repository!\n",
+    "\n",
+    "Maybe you are thinking: \"why should I use Ibis?\". Well, if you have any of these follow issues, probably you should consider use ibis in your analytical workflow!\n",
+    "\n",
+    "- if you need to get data from a SQL Database but you don't know much about SQL ...\n",
+    "- if you create SQL statements manually using string and have a lot of `IF`'s in your code that compose specific parts of your SQL code (it could be pretty hard to maintain and it will makes your code pretty ugly) ...\n",
+    "- if you need to handle data with big volume ..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<!-- TEASER_END -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "If you want to learn more about ibis consider to take a look into these tutorials:\n",
+    "\n",
+    "* https://docs.ibis-project.org/tutorial.html\n",
+    "\n",
+    "Do you want to watch some interesting video about Ibis? Check this out:\n",
+    "\n",
+    "* https://www.youtube.com/embed/8Tzh42mQjrw?start=1625\n",
+    "\n",
+    "**Now, let's check out some work developed here at Quansight in the last months!**\n",
+    "\n",
+    "During the last months OmniSci and Quansight were working together to add a backend on Ibis for OmniSciDB (former MapD) that also resulted on new expressions/operators on Ibis core, such as:\n",
+    "\n",
+    "- GeoSpatial data types and operations\n",
+    "- Trigonometric operations\n",
+    "- Some statistcal operations"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, let's connect to a *OmniSciDB* and play with this new features!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# install the dependencies if you need!\n",
+    "# !conda install -y ibis-framework=1.1.0 pyarrow pymapd vega geopandas geoalchemy2 shapely matplotlib --force-reinstall"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "ibis: 1.1.0\n"
+     ]
+    }
+   ],
+   "source": [
+    "import ibis\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "print('ibis:', ibis.__version__)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "omniscidb_cli = ibis.mapd.connect(\n",
+    "    host='metis.mapd.com', \n",
```

Comment:
is this a test instance that will work for readers? either way, it needs a comment I think